login-workflow/src/new-architecture/screens/VerifyCodeScreen/VerifyCodeScreen.test.tsx

```diff
@@ -0,0 +1,184 @@
+import React from 'react';
+import '@testing-library/jest-dom';
+import { cleanup, render, screen, fireEvent, RenderResult } from '@testing-library/react';
+import { VerifyCodeScreen } from './VerifyCodeScreen';
+import { VerifyCodeScreenProps } from './types';
+import {
+    RegistrationContextProvider,
+    RegistrationContextProviderProps,
+    i18nRegistrationInstance,
+} from '../../contexts';
+import { RegistrationWorkflow } from '../../components';
+
+const defaultProps: RegistrationContextProviderProps = {
```

Comment:
You can use the existing defaultProps:
`import { defaultProps as registrationContextProviderProps } from '../../contexts/RegistrationContext/RegistrationContextProvider.test';`